### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by sserwangaedrisa**
HERE ARE THE CHANGES MADE IN THIS COMMIT
1. Writing the html code without the navigation bar
2. Styling the written html using the style.css file
3. Adding a javascript file (style.js) that allows the users to add and remove the books from the library
4. Styling the added books list to match the found books

### CODE REVIEW

**Review by arnoldnekemiah — CHANGES_REQUESTED** [review]
You have done alot to present your work the best way you can but their are a number of changes you have to make to complete this milestone. :cry: 

1.Next time when creating a pull request, you name it according to the project that you are doing.FOr every new milestone of the project you will create a new branch.For now You can keep this thread.
2. Code not presented to standard. Click here to learn how to configure linters into your project, and learn how to use them to correct your work syntax.
3. The approach you have used to do this activity is wrong.
Below is a list of Links to teach you how Object-method works 

- [Object literal notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)
- [Object() constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/Object).
- [Object.prototype.constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor).
- [Object.create()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create).
[Object.assign()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign).

4. Also consider learning how Local storage works.

Please try again and submi this very pull request.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+#contact-head{
+    text-align: center;
+}
+#contact-details{
+   text-align:left;
+    padding-left:15%;
+}
+#ul2{
+    border: none;
+}
```
[the remark this answers is not in the document]
Comment (sserwangaedrisa) [reply style.css]: I did not manage using linters but i used another code formatter called prettier to try solving that issue

File: style.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+var bookSection=document.getElementById('ul')
+
+
+// for (var i=0;i<remove.length;i++){
+//     remove[i].addEventListener('click', function(event){
+//         // console.log(Array.from(remove).indexOf(event.target))
+//         var listIndex=Array.from(remove).indexOf(event.target)
+//         listItem[listIndex].parentNode.removeChild(listItem[listIndex])
+//     })
+// }
```
[the remark this answers is not in the document]
Comment (sserwangaedrisa) [reply style.js]: i removed all the commented code from my  work

File: style.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+
+var remove= document.getElementsByClassName('removeButton')
+var add= document.getElementById('addingButton')
+var listItem=document.getElementsByClassName('list')
+var bookSection=document.getElementById('ul')
+
```
[the remark this answers is not in the document]
Comment (sserwangaedrisa) [reply style.js]: i minimized the usage of var. And now i know the difference between the two words

File: style.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+//         // console.log(Array.from(remove).indexOf(event.target))
+//         var listIndex=Array.from(remove).indexOf(event.target)
+//         listItem[listIndex].parentNode.removeChild(listItem[listIndex])
+//     })
+// }
+var title=document.getElementById('Title')
+var author=document.getElementById('Author')
+
+add.addEventListener('click', function(){
+    // console.log(title.value)
```
[the remark this answers is not in the document]
Comment (sserwangaedrisa) [reply style.js]: removed

**Pull request closed without merging**